src/components/Plugin/ContributionRewardExtRewarders/Competition/Details.tsx

```diff
@@ -443,6 +443,7 @@ export default withSubscription({
     ${CompetitionSuggestion.fragments.CompetitionSuggestionFields}
     `;
 
+    // eslint-disable-next-line no-console
```

Comment:
lets delete this console.log?

Reply:
Meaning there is no need in the `cacheQuery` from graphQL? this `console.log` is the only one who uses this.

Reply:
Oh yeah, this reminds me that I've looked at this before.  @jellegerbrandy added the `console.log` and commented out the code that actually executes the query.  I don't know why.  Jelle?